src/components/time-on-target-chart/TimeOnTargetChart.svelte

```diff
@@ -31,63 +31,65 @@
     const hideDelay = (): void => {
 	    haveGaussSelected = Object.values($selectedWeapons).filter(weapon => weapon.weaponName === 'gausscannon').length > 0;
     };
+
+    const chartToImage = (): void => {
+        domtoimage.toBlob(document.getElementById('customChart'))
+        .then((blob) => {
+            window.saveAs(blob, 'My_Chart.png');
+        });
+    };
 </script>
 
 
 <div class="time-on-target-chart">
-    <div class="mb-3 ml-2">
-        <h1 class="mt-1 mb-0 has-text-centered">Create a Chart</h1>
-
-        <h3 class="mt-2 mb-0">Name your ship!</h3>
-        <input type="text" bind:value={shipName} maxlength="26" class="text-input large-text-input p-0 has-text-centered" placeholder="Ship Name Here">
-
-        <h3 class="mt-2 mb-0">Ammo Selection</h3>
-        <select class="dropdown-select mt-1" bind:value={ammo}>
-            <option value="Basic">Basic</option>
-            <option value="Standard">Standard</option>
-            <option value="Premium">Premium</option>
-        </select>
-
-        <h3 class="mt-2 mb-0">Accuracy</h3>
-        <select class="dropdown-select mt-1" bind:value={accuracy}>
-            <option value="100">100%</option>
-            <option value="75">75%</option>
-            <option value="50">50%</option>
-        </select>
-
-        <label class="toggle-mechs">
-            <input type=checkbox bind:checked={toggleMm}>
-            Toggle Millimechs
-        </label>
-
+    <div class="chart-settings">
+        <div class="mb-3 ml-2">
+            <h1 class="mt-1 mb-0 has-text-centered">Create a Chart</h1>
+    
+            <h3 class="mt-2 mb-0">Name your ship!</h3>
+            <input type="text" bind:value={shipName} maxlength="26" class="text-input large-text-input p-0 has-text-centered" placeholder="Ship Name Here">
+    
+            <h3 class="mt-2 mb-0">Ammo Selection</h3>
+            <select class="dropdown-select mt-1" bind:value={ammo}>
+                <option value="Basic">Basic</option>
+                <option value="Standard">Standard</option>
+                <option value="Premium">Premium</option>
+            </select>
+    
+            <h3 class="mt-2 mb-0">Accuracy</h3>
+            <select class="dropdown-select mt-1" bind:value={accuracy}>
+                <option value="100">100%</option>
+                <option value="75">75%</option>
+                <option value="50">50%</option>
+            </select>
+    
+            <button class="download-button" on:click={chartToImage}>Download Chart <br> 
+                                                                        as PNG</button>

```

Comment:
Why the new line here? Just keep in on the same line...

There is also no need for the blank line under this

Reply:
Fixed